### DISCUSSION TITLE
Alpha 184

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#7203] [Release Notes]
- d70d1b2: [Updated xterm to v5](https://github.com/xtermjs/xterm.js/releases/tag/5.0.0) - fixes #5612
- 6fff872: Support private key forwarding to winscp (Artur) [#7143](https://github.com/Eugeny/tabby/pull/7143)
- 6fbb2a2: Added Linux ARM builds (Jai A P) [#6612](https://github.com/Eugeny/tabby/pull/6612)
- 166fc2e: Fixed outdated plugins in the plugin list (Thomas LACAZE) [#7021](https://github.com/Eugeny/tabby/pull/7021)
- 753667a: Add Melange (Dark) theme (Spenser Black) [#7158](https://github.com/Eugeny/tabby/pull/7158)
- 3090686: Change the default "scroll up" / "scroll down" hotkeys - fixes #6878
- 27e023a: Added PgUp/PgDown support and fuzzy search to the profile selector - fixes #6872
- fbea7db: Expose combined context menu in tab header and add visual context menu button - fixes #6966
- 2bf0ef3: Only trim pasted text when it's a single line - fixes #7028, fixes #7064

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.184'>Alpha 184</a>.</em>

**Comment by frameyl** [comment]
On my windows10 (2021Q2), the right click paste and middle-click paste do not work any more.
I have to revert to version 183 to solve the issue.
Anyone has the same problem as me?

**Reply by VirtualLG** [reply]
I have got the same problem. And also the mouse select in vim, mouse operation is disabled in vim, is not working in this version.
I alse noticed that the ssh session slagging while typing.
All these things woking fine in `Alpha 183`.
